easylist_cookie/easylist_cookie_international_specific_block.txt

```diff
@@ -7,6 +7,7 @@
 ||consent.berliner-kurier.de^
 ||consent.faz.net^
 ||consent.gutefrage.net^
+||consent-wrapper.jameda.de^
```

Comment:
I wonder if it would be more optimal to add the cloudfront domain `d2eucw4fxv7nil.cloudfront.net.`

```
drill consent-wrapper.jameda.de @192.168.1.2
;; ->>HEADER<<- opcode: QUERY, rcode: NOERROR, id: 25649
;; flags: qr rd ra ; QUERY: 1, ANSWER: 5, AUTHORITY: 0, ADDITIONAL: 0 
;; QUESTION SECTION:
;; consent-wrapper.jameda.de.   IN      A

;; ANSWER SECTION:
consent-wrapper.jameda.de.      864     IN      CNAME   d2eucw4fxv7nil.cloudfront.net.
d2eucw4fxv7nil.cloudfront.net.  864     IN      A       13.33.240.36
d2eucw4fxv7nil.cloudfront.net.  864     IN      A       13.33.240.116
d2eucw4fxv7nil.cloudfront.net.  864     IN      A       13.33.240.82
d2eucw4fxv7nil.cloudfront.net.  864     IN      A       13.33.240.114

drill consent-wrapper.jameda.de @192.168.1.53
;; ->>HEADER<<- opcode: QUERY, rcode: NOERROR, id: 4852
;; flags: qr rd ra ; QUERY: 1, ANSWER: 5, AUTHORITY: 0, ADDITIONAL: 0 
;; QUESTION SECTION:
;; consent-wrapper.jameda.de.   IN      A

;; ANSWER SECTION:
consent-wrapper.jameda.de.      240     IN      CNAME   d2eucw4fxv7nil.cloudfront.net.
d2eucw4fxv7nil.cloudfront.net.  1       IN      A       143.204.238.39
d2eucw4fxv7nil.cloudfront.net.  1       IN      A       143.204.238.22
d2eucw4fxv7nil.cloudfront.net.  1       IN      A       143.204.238.29
d2eucw4fxv7nil.cloudfront.net.  1       IN      A       143.204.238.124
```

As it could appear to simply be a redirection, hence probably used by more domains:

<details><summary>note to self(@spirillen)</summary>

<https://www.mypdns.org/T4129>

</details>